src/components/AnnotationList/AnnotationList.scss

```diff
@@ -0,0 +1,23 @@
+@import '../../variables.scss';
+
+.ba .ba-annotation-list {
+    padding-bottom: 10px;
+    width: 100%;
+
+    .ba-annotation-list-item {
```

Comment:
Please try to avoid nesting when possible to keep specificity low. Let's make this a goal for all code going forward.